components/esm/loadweap.hpp

```diff
@@ -75,5 +81,34 @@ struct Weapon
     void blank();
     ///< Set record to default state (does not touch the ID).
 };
+
+struct WeaponType
```

Comment:
This struct is representation of a new record type (although it isn't exactly implemented in full right now). I'm not sure if it's wise to put it into the technically unrelated weapon record loading header if it will have to be moved anyway.

Reply:
I see no point to put it to the separate file. As you said, there is so such standalone ESM record yet.